store/views.py

```diff
@@ -77,6 +99,42 @@ def loanBookView(request):
 @csrf_exempt
 @login_required
 def returnBookView(request):
-    pass
+    post_data = request.POST
+    try:
+        bookCopy = BookCopy.objects.get( id__exact=post_data['bid'], borrower=request.user )
+        message = 'success'
+        bookCopy.status = True
+        bookCopy.borrower = None
+        bookCopy.save()
+    except:
+        message = 'failure'
+    
+    response_data = {
+        'message': message
+    }
+    return JsonResponse(response_data)
 
+@csrf_exempt
+@login_required
+def rateBookView(request):
+    post_data = request.POST
+    book = get_object_or_404(Book, pk=post_data['bid'])
+    response = {'message':'failure'}
+    if post_data['rating']>10 or post_data['rating']<0:
+        return JsonResponse(response)
+    try :
+        new_rating = BookRating.objects.get(user=request.user, book=book)
+        new_rating.rating = post_data['rating']
+    except:
```

Comment:
As a good coding practice, whenever you use try-except block, capture only the exceptions which you want to catch (IndexError, IntegrityError, etc.)